### ISSUE TITLE
Cannot view facet plot

### ISSUE THREAD

**Issue by ElisonSherton** [shaunirwin/facets_example#1]
Hello @shaunirwin 

Thanks for this tutorial. I have been following this tutorial and [this notebook](https://www.kaggle.com/davidwagnerkc/visualizing-33k-images-with-google-facets) for setting up a dashboard to visualize my data. 

As you have mentioned in your tutorial, I have installed all the requirements and created metadata and my own sprite_atlas image and trying to use your code to visualize the same in my notebook as shown below.

![image](https://user-images.githubusercontent.com/45599762/108799938-c042cb80-75b7-11eb-88ed-5540f9cf6a5b.png)

Also, I have locally kept the facets directory by git cloning it from their official source code. 

Despite all this, I am still not able to visualize the plots using the facets library, the sprite atlas file is 15 MB big so can't attach here but it's proper I checked.... 

Can you please help me debug this issue? I would be highly obliged if you could help me out here.

Thanks & Regards,
Vinayak Nayak.

**Comment by shaunirwin** [comment]
Hi @ElisonSherton, which browser are you using? I haven't tried it recently, but previously I found that it worked in Google Chrome, not Firefox. In Firefox it didn't give any error, but simply did not display anything.

**Comment by ElisonSherton** [comment]
Hey @shaunirwin,

Thanks for the prompt reply! I found this [colab notebook](https://colab.research.google.com/github/PAIR-code/facets/blob/master/facets_dive/colab_dive_atlas_demo.ipynb#scrollTo=Ck1Dcat_pnv5) later and used the code mentioned therein and the visualization works fine now.

![image](https://user-images.githubusercontent.com/45599762/108836718-31ee3a00-75f7-11eb-95a4-1974ef68eb1a.png)

So basically, using an extra script tag and encoding the image in binary format in the html layout did the trick for me as can be seen in the snipper above.

Thanks & Regards,
Vinayak.
